tests/src/Exception/AmbiguousWysiwygEditorException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\joinup\Exception;
+
+/**
+ * Exception thrown when multiple WYSIWYG editors with the same label are found.
+ */
+class AmbiguousWysiwygEditorException extends \Exception {
```

Comment:
For the record, I don't see any real help of defining extra exceptions that provide no real help. We only show a message and we only grab the message when an error is occurring. Splitting this up into multiple exceptions without any functionality behind is just creating new files.. You could even have a generic `BehatException` for all that matters if for any reason we want to have a separate Exception class.

Reply:
What you describe is what we do in step definitions and in that case I fully agree. Step definitions are the end point and nothing is wrapping it that can catch the exceptions.

But in traits they are very useful because these are called from other code that can catch the exceptions to take a different logic path, or provide a better error message. This function `::getWysiwyg()` that throws it is something I we will use in a lot of different steps now that we start working with paragraphs.

The main reason I put it in though is to be consistent with `WysiwygEditorNotFoundException` which is thrown in the same method. It seems strange to throw a specific exception only for the case where the editor is not found, but a generic one when there are multiple matching ones.